docs/contributing/contributing.md

```diff
@@ -57,10 +62,9 @@ Based on the current `upstream/development` (or `upstream/docs/development`) cha
         - Follow the [Structure Guide](structure-guide.md)
         - Compare your code style to the one used all over JDA and ensure you
           do not break the consistency (if you find issues in the JDA style you can include and update it)
-
+     
```

Comment:
Was this added for a reason? or was that just an oopsie

Reply:
Probs GitHub Desktop messing around